js/main.js

```diff
@@ -1,58 +1,106 @@
-/* eslint-disable no-use-before-define */
-// Create array of books
-const bookTitle = document.querySelector('#title');
-const bookAuthor = document.querySelector('#author');
-const bookAdd = document.querySelector('#available-books');
-const btn = document.querySelector('#add-button');
-
-if (!localStorage.getItem('bookInfo')) {
-  localStorage.setItem('bookInfo', JSON.stringify([]));
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable **all** linter checks on your code to ensure that your code is robust code and clean.